components-core/src/main/java/com/adyen/checkout/components/core/PaymentMethodTypes.kt

```diff
@@ -36,6 +36,7 @@ object PaymentMethodTypes {
     const val GOOGLE_PAY_LEGACY = "paywithgoogle"
     const val IDEAL = "ideal"
     const val MB_WAY = "mbway"
+    const val MEAL_VOUCHER_FR = "mealVoucher_FR" // not a txVariant, only to be used for /payments call
```

Comment:
In that case in there any benefit for merchants to have this as a public value?

Reply:
Good point. Restricted the access to library group.